### PR TITLE
Fixes to FSI case for adding the tower

### PR DESCRIPTION
**Pull request by psakievich**

### CODE REVIEW

File: 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml [hunk 1]
```diff
@@ -154,6 +158,12 @@ realms:
       use_wall_function: no
       turbulent_ke: 0.0
 
+  - wall_boundary_condition: bc_wall
+    target_name: tower-wall
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
```
Comment (ashesh2512) [inline 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml:164]: We need this too - 
  - symmetry_boundary_condition: bc_ground
    target_name: tower-ground
    symmetry_user_data:

File: 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml [hunk 2]
```diff
@@ -178,15 +189,14 @@ realms:
...
-    mesh_parts:
-      - blade1-HEX
-      - blade2-HEX
-      - blade3-HEX
+  - name: tower_translate
+      - tower-fluid-HEX
     motion:
       - type: translation
-        displacement: [1772.54, 2080.0, 90.0]
+        displacement: 1800.0, 1800.0, 90.0]
```
Comment (ashesh2512) [inline 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml]: I think this should just be rotor position ([1772.54, 2080.0, 90.0]) + (5, 0, 0). This also needs to be consistent with inp.yaml

**Review by ashesh2512 — APPROVED** [review]

**Comment by ashesh2512** [comment]
As a sanity check we should do `ncdump -v hub_ref_pos <turbine output netcdf file>` after the `openfastcpp sim` to make sure we have the correct rotor position before setting off the turbine run.

File: 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml [hunk 3]
```diff
@@ -164,7 +176,8 @@ realms:
           - blade1-HEX
           - blade2-HEX
           - blade3-HEX
-        wall_parts: [blade1, blade2, blade3]
+          - tower-fluid-HEX
+        wall_parts: [blade1, blade2, blade3, tower-fluid-HEX]
         ovset_parts: [outerBC]
```
Comment (psakievich) [inline 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml]:
```suggestion
        ovset_parts: [outerBC, tower-overset]
```

File: 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml [hunk 4]
```diff
@@ -164,7 +176,8 @@ realms:
           - blade1-HEX
           - blade2-HEX
           - blade3-HEX
-        wall_parts: [blade1, blade2, blade3]
+          - tower-fluid-HEX
+        wall_parts: [blade1, blade2, blade3, tower-fluid-HEX]
```
Comment (psakievich) [inline 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml]:
```suggestion
        wall_parts: [blade1, blade2, blade3, tower-wall]
```

File: 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml [hunk 5]
```diff
@@ -178,15 +191,14 @@ realms:
...
         centroid: [0.0, 0.0, 0.0]
+      - type: translation
+        displacement: [1772.54, 2080.0, 90.0]
         
-  - name: turbine_translate
-    mesh_parts:
-      - blade1-HEX
-      - blade2-HEX
-      - blade3-HEX
+  - name: tower_translate
```
Comment (psakievich) [inline 1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml:197]:
```suggestion
  - name: tower_translate
     mesh_parts:
```

**Comment by psakievich** [comment]
Merging since this is what I ran on frontier. I am going to clean up now in a follow on PR

**Pull request merged**